index.html

```diff
@@ -180,6 +180,29 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
   </div>
 </template>
 
+<script type="text/javascript">
+  var fireballSize = 22;
+
+  var getFireballSpeed = function (isLeftWind) {
+    return isLeftWind ? 2 : 5;
+  };
+  
+  var wizardSpeed = 3;
+  
+  var wizardWidth = 70;
+  
+  var getWizardHeight = function() {
+    return 1.337 * wizardWidth;
+  };
+  
+  var getWizardX = function (FieldSizeX) {
```

Comment:
```suggestion
  var getWizardX = function (fieldSizeX) {
```
Согласно требования интенсива, необходимо называть переменные со строчной буквы.

От себя могу добавить, что ожидаю имя класса, когда виду наименование переменной с заглавной буквы.

Reply:
Принято.
Ещё не прочёл критерии, но задача стоит.

Где прочитать про классы?

Reply:
Классы - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes. 
Скорее всего, пока рано в них углубляться, но общее представление не помешает. 
В текущем интенсиве мы не будем их использовать.